modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -403,6 +446,80 @@ Users who can access the Windows Network Drive storage via share won't receive a
 
 NOTE: `wnd.activity.sendToSharees` key depends on the `wnd.activity.registerExtension` key to take effect.
 
+=== Collaborative WND
+
+CWND can only be set by an admin in menu:Settings[Admin > Storage]. This mount type cannot be selected by users in the user section. To prepare access for your mount point using the CWND mount type, you must provide a _Service Account_ (SA) which is an oridnary SMB user granting read access to the share you want to mount. You can use one SA for all CWND mounts or separate ones. The SA is used to gather the contents of a share, used by the WND Listener and provides a common `file_id` to all accessing users, while the accessing user has only access to those files and folders he is granted for.
+
+. As an admin, goto menu:Settings[Admin > Storage] and create a new CWND based mount point.
++
+.Figure 5. Add a Collaborative Windows Network Drive Mount
+image:enterprise/external_storage/windows_network_drive/cwnd_add_storage.png[Add Collaborative Windows Network Drive Mount, width=50%]
+. Chose any name for the mount point that fits your needs.
+. Select User Login Type.
++
+[IMPORTANT]
+====
+Only the following three are valid selections for CWND
```

Comment:
Technically, all of them are valid, but it's true that some of them don't have sense to be "collaborative".
Both global credentials and username+password don't have sense to be "collaborative". The credentials hardcoded in config file can also fit as "valid" here, although the use case is pretty limited from my point of view, so maybe we can skip it too.

Reply:
What about to rephrase this line to highlight that those are working AND sensful ones, any suggestions?

Reply:
Yes, that's fine

Reply:
Suggestion see: https://github.com/owncloud/docs/pull/3529#discussion_r627472206